test/controllers/custom_registrations_controller_test.rb

```diff
@@ -35,6 +35,20 @@ class CustomRegistrationsControllerTest < Devise::ControllerTestCase
     assert @controller.update_block_called?, "update failed to yield resource to provided block"
   end
 
+  test "yield resource to block on destroy failure" do
+    sign_in @user
+    delete :destroy, params: { user: { } }
+    assert @controller.destroy_block_called?, "destroy failed to yield resource to provided block"
+  end
+
+  test "yield resource to block on destroy success" do
+    swap Devise, require_password_to_destroy: true  do
```

Comment:
This spec is falling on Rails 4.1 and 4.2. I ran this test without this `swap` and it worked on Rails 4.2 - I don't know why 🙈. Maybe we don't need this here.